test/phoenix_live_view/integrations/live_reload_test.exs

```diff
@@ -12,12 +12,6 @@ defmodule Phoenix.LiveView.LiveReloadTest do
   @endpoint Helpers.Endpoint
   @pubsub Phoenix.LiveView.PubSub
 
-  setup_all do
-    {:ok, _} = Application.ensure_all_started(:phoenix_live_reload)
-
-    :ok
-  end
-
```

Comment:
By looking at past commits it seemed that this was a previous attempt at fixing this test failure. So I suspect not necessary any more. Let me know if you think otherwise :+1: